docs/ansible/roles/debops.postldap/guides.rst

```diff
@@ -28,7 +28,99 @@ email addresses. The default configuration uses first the aliases set by
 See also :ref:`debops.dovecot` and :ref:`debops.roundcube` for an IMAP server
 and Email-Webclient correspondingly.
 
+The following example shows a real-world™ setup on the Hetzner Cloud. It consists of two servers,
+one ``controller`` and a ``mail-server``. The LDAP directory is hosted also in the ``controller``.
+``mail-server`` has access to LDAP over an internal network (10.10.10.0/24) attached directly to the VMs.
+This setup has no internal DNS server (no split-DNS), thus internal IPs are mapped to DNS entries in the form ``$service.hetzner.mydomain.net``.
+See `secret__ldap_fqdn`.
+
+.. code-block:: yaml
+  :caption: ansible/inventory/group_vars/hetzner/ldap.yml
```

Comment:
This. Changes. Everything.

Reply:
What do you have in mind.
I thought of a "simple" tutorial/example on how postfix and other services can be configured (copy-pasted) in order to have a working solution in no time.

Reply:
The default configuration in roles is supposed to give you a working solution - DebOps is designed for use in production, you are supposed to be able to set up new hosts, apply the playbook and go, all of the role default variables are there so that you can tweak stuff from a typical use case to a more specific one. If that's not the case with some roles that are supposed to configure stuff automatically, it needs to be fixed in the roles, not provided as some kind of additional tutorial with required configuration changes.

But in this specific case I was referring to the `:caption:` option in reStructuredText. I would like to be able to label various inventory examples with specific paths, but when I tried out `:caption:` with the current RTD theme it was kinda meh - the path was centered in kind of away from the file snippet. It could look better, IMO.